fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -2342,7 +2362,8 @@ public void handleDMLStmt(ExecPlan execPlan, DmlStmt stmt) throws Exception {
                                         "or set parallel_fragment_exec_instance_num to a lower value in session variable");
                     } else {
                         ErrorReport.reportTimeoutException(ErrorCode.ERR_TIMEOUT, getExecType(), timeout,
-                                "please increase the 'insert_timeout' session variable and retry");
+                                String.format("please increase the '%s' session variable and retry",
+                                        SessionVariable.INSERT_TIMEOUT));
```

Comment:
Does the timeout depend on isExecLoadType()

Reply:
only insert is used here.